OpenRA.Mods.Common/Traits/World/MapOptions.cs

```diff
@@ -41,7 +41,7 @@ public class MapOptionsInfo : TraitInfo, ILobbyOptions, IRulesetLoaded
 		public readonly string TechLevelDropdownLabel = "Tech Level";
 
 		[Desc("Tooltip description for the tech level option in the lobby.")]
-		public readonly string TechLevelDropdownDescription = "Change the units and abilities at your disposal";
+		public readonly string TechLevelDropdownDescription = "The units and abilities at players' disposal";
```

Comment:
Nitpicking: If we're sticking an apostrophe on the end of `players` here - should we do the same in other text where it's used as a possessive?

Reply:
:thinking: This might be the only place where a possessive is used for "players". I don't know about other similar words. I'm all for consistency but I'm not a native speaker so I can't decide what is better.

Reply:
I think you're right here and I'm not enough of a language nerd to be sure on the other usages. Let's just roll with it.